cmd/semdiffstat/main.go

```diff
@@ -79,3 +82,14 @@ func pluses(ins int) string {
 func minuses(del int) string {
 	return boldRed.Sprint(strings.Repeat("-", del))
 }
+func alignLines(changes []*semdiffstat.Change) (n int) {
+	var lens int
+	for _, c := range changes {
+		lens = c.InsLines + c.DelLines
+		lens = len(strconv.Itoa(lens))
```

Comment:
If we called the return value `max`, we could use `n` here. And no need to declare at the start of the function. So instead:

```
n := len(strconv.Itoa(c.InsLines+c.DelLines))
if n > max {
  max = n
}
```